src/main/java/pl/pjwstk/woloappapi/controller/OrganisationController.java

```diff
@@ -0,0 +1,41 @@
+package pl.pjwstk.woloappapi.controller;
+
+import lombok.AllArgsConstructor;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+import pl.pjwstk.woloappapi.model.Organisation;
+import pl.pjwstk.woloappapi.service.OrganisationService;
+
+import java.util.List;
+
+
+@RestController
+@AllArgsConstructor
+@RequestMapping("/organisation")
+public class OrganisationController {
+
+    private final OrganisationService organisationService;
+
+    @GetMapping("/all")
+    public ResponseEntity<List<Organisation>> getOrganisations(){
+        List<Organisation> Organisations = organisationService.getAllOrganisations();
+        return new ResponseEntity<>(Organisations, HttpStatus.OK);
+    }
+
+    @GetMapping("/{id}")
+    public ResponseEntity<Organisation> getOrganisationById(@PathVariable Long id){
+        return new ResponseEntity<>(organisationService.getOrganisationById(id), HttpStatus.OK);
+    }
+    @PostMapping("/add")
+    public ResponseEntity<HttpStatus> addOrganisation(@RequestBody Organisation Organisation){
+        organisationService.createOrganisation(Organisation);
+        return new ResponseEntity<>(HttpStatus.CREATED);
+    }
+    @PostMapping("/delete/{id}")
+    public ResponseEntity<HttpStatus> deleteOrganisation(@PathVariable Long id){
+        organisationService.deleteOrganisation(id);
+        return new ResponseEntity<>(HttpStatus.OK);
+    }
+
+}
```

Comment:
w poprzednich klasach bywała pusta linia, a czasem więcej niż 1 na końcu. Tu i w niektórych innych nie ma. Zawsze tak samo. W każdej klasie.

BTW, to powinno być robione automatycznie. Np. w pre-commit dla githuba. Są do tego automaty.